pkg/pb/notifications.proto

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package notifications;
+
+import "gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+message Notification{
+    enum Scope {
+        CUSTOM = 0;
+        AUDIT = 1;
+        UPTIME = 2;
+        REPAIR = 3;
+        DISQUALIFICATION = 4;
+        GRACEFUL_EXIT = 5;
+        VETTING = 6;
+    }
+
+    enum Level {
+        INFO = 0;
+        WARN = 1;
+        ERROR = 2;
+    }
+
+    Scope scope = 1;
+    Level level = 2;
+    repeated string tags = 3;
+    string message = 4;
```

Comment:
i think this protobuf should be the most rare - satellite operator messages. if we have messages that are common, the message should be chosen by the storage node in the SNO's language, and not by the satellite operator. routine messages should have some sort of enum type, and this message field should only be used for exceptional circumstances only